test/cmd/lfstest-gitserver.go

```diff
@@ -838,6 +859,13 @@ func (c LocksByCreatedAt) Swap(i, j int)      { c[i], c[j] = c[j], c[i] }
 var lockRe = regexp.MustCompile(`/locks/?$`)
 
 func locksHandler(w http.ResponseWriter, r *http.Request) {
+	repo, err := repoFromLfsUrl(r.URL.Path)
```

Comment:
Good call. There was one spot that `locksHandler()` was called outside of `lfsHandler()`, but that was the old global `/locks` route that's unneeded. 🔥